tests/test_cluster/test_elbow.py

```diff
@@ -400,6 +395,32 @@ def test_timings(self):
 
         self.assert_images_similar(visualizer)
 
+    def test_sample_weights(self):
+        """
+        Test that passing in sample weights correctly influences the clusterer's fit
+        """
+        seed = 1234
+
+        # original data has 5 clusters
+        X, y = make_blobs(
+            n_samples=[5, 30, 30, 30, 30],
+            n_features=5,
+            random_state=seed,
+            shuffle=False,
+        )
+
+        visualizer = KElbowVisualizer(
+            KMeans(random_state=seed), k=(2, 12), timings=False
+        )
+        visualizer.fit(X)
+        assert visualizer.elbow_value_ == 5
+
+        # weights should push elbow down to 4
+        weights = np.concatenate([np.ones(5) * 0.0001, np.ones(120)])
+
+        visualizer.fit(X, sample_weight=weights)
+        assert visualizer.elbow_value_ == 4
```

Comment:
Excellent test &mdash; also a good, clear example of why the sample weights can have such a big impact on the clusters! 

_Note: Other than autoformatting, I've removed the calls to `finalize` here, since that just adds the finishing touches to the visualizer, like title, etc, so we don't need it in this case & it makes the tests a teensy bit faster to run if we can avoid it!_